### ISSUE TITLE
Potentially 2 LR Schedulers used for Object Detection Tutorial

### ISSUE THREAD

**Issue by JoanneLin168** [pytorch/tutorials#2631]
Link to tutorial: https://pytorch.org/tutorials/intermediate/torchvision_tutorial.html

Hi there, I am following the object detection tutorial and have copied the necessary files from `pytorch/vision/main/references/detection/`. When inspecting the `engine.train_one_epoch()` function, I see that it creates a LR scheduler (LinearLR) there and calls `lr_scheduler.step()` inside `train_one_epoch()`, however in the tutorial it also creates its own LR scheduler (StepLR) in the main script, and also calls `lr_scheduler.step()` within the training loop.

Is this meant to be the case or should the tutorial be updated?
